tests/test_backend.py

```diff
@@ -2932,6 +2933,27 @@ def func(input_holder):
         for input_val in input_vals:
             self._run_test_case(func, [_OUTPUT], {_INPUT: input_val})
 
+    @skip_tf_cpu("only tf_gpu can run CudnnGPU")
+    @check_opset_min_version(11, "GRU")
+    def test_cudnngru(self):
+        def func():
+            seq_length = 3
+            batch_size = 5
+            input_size = 2
+            num_layers = 2
+            num_units = 2
+            num_dirs = 2
+            initializer = init_ops.constant_initializer(0.5)
+            x = np.random.randint(0, 100, [seq_length, batch_size, input_size]).astype(np.float32)
```

Comment:
You want random badly to be outside of func() because if you run this against tf and than onnxruntime they will be different. Better to pass them in arguments to func()